### ISSUE TITLE
Passed validation locally, but not online

### ISSUE THREAD

**Issue by sheepzh** [mozilla/addons-linter#3656]
### Describe the problem and steps to reproduce it:

I can pass the validation locally with addons-linter, but not in the store while publishing.

### What happened?

I packaged my addon, and validated it with addon-linter.
```
addons-linter --version
>>> 3.1.0
addons-linter [my.zip]
```
The local validation passed.

```
验证概要：

errors          0              
notices         0              
warnings        25   
```
Then I publish the addon in the store, one error occurred.
```
常规测试
1 个错误, 0 个警告, 0 个提醒
Sorry, we couldn't load your WebExtension.

错误: Validation was unable to complete successfully due to an unexpected error.

Check https://developer.mozilla.org/en-US/Add-ons/WebExtensions to ensure your webextension is valid or file a bug at http://bit.ly/1POrYYU
```

### What did you expect to happen?

Maybe, I think, I can pass the online validation. It's the first time to fail for this addon.

### Anything else we should know?

You can clone the repo and build one addon file. 
```git
git clone https://github.com/sheepzh/make-zero.git
git checkout v1.5.2
npm install 
npm run build:prod
addons-linter market_packages/make-zero-1.5.2.zip
```
You can also load the addon from the directory <u><i>chrome_dir</i></u>  after building.

**Comment by willdurand** [comment]
Hi, we have identified an issue on our servers, which prevents the submission process. We are working on fixing it. Sorry.

**Comment by wagnerand** [comment]
This should have been addressed now. Sorry for the inconvenience.

**Comment by sheepzh** [comment]
> This should have been addressed now. Sorry for the inconvenience.

Thanks for your hard work !
